libs/mainnet-stats-manager/src/components/good-threshold-indicator/good-threshold-indicator.tsx

```diff
@@ -0,0 +1,21 @@
+import { value, goodThreshold } from '../../config/types';
+
+interface GoodThresholdIndicatorProps {
+  goodThreshold: goodThreshold | undefined;
+  value: value;
+}
+
+export const GoodThresholdIndicator = ({
+  goodThreshold,
+  value,
+}: GoodThresholdIndicatorProps) => {
+  return (
+    <div
+      className={`inline-block w-8 h-8 mb-[0.15rem] mr-8 rounded ${
+        (goodThreshold &&
+          (goodThreshold(value) ? 'bg-intent-success' : 'bg-intent-danger')) ||
```

Comment:
I don't see these classes used or imported anywhere else?

Reply:
They're in the theme, I think this is the intended use for them